clients/privacy-center/pages/index.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState } from 'react';
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import {
+  Flex,
+  Heading,
+  Text,
+  Stack,
+  Box,
+  Alert,
+  AlertIcon,
+  AlertDescription,
+  CloseButton,
+} from '@fidesui/react';
+import Image from 'next/image';
+
+import { useRequestModal, RequestModal } from '../components/RequestModal';
+import type { AlertState } from '../types/AlertState';
+
+import config from '../config/config.json';
+
+const Home: NextPage = () => {
+  const [alert, setAlert] = useState<AlertState | null>(null);
+  const { isOpen, onClose, onOpen, openAction } = useRequestModal();
+  return (
+    <div>
+      <Head>
+        <title>Privacy Center</title>
+        <meta name="description" content="Privacy Center" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <header>
+        <Flex
+          bg="gray.100"
+          minHeight={14}
+          p={1}
+          width="100%"
+          justifyContent="center"
+          alignItems="center"
+        >
+          {alert ? (
+            <Alert
+              status={alert.status}
+              minHeight={14}
+              maxWidth="5xl"
+              zIndex={1}
+              position="absolute"
+            >
+              <AlertIcon />
+              <AlertDescription>{alert.description}</AlertDescription>
+              <CloseButton
+                position="absolute"
+                right="8px"
+                onClick={() => setAlert(null)}
```

Comment:
Again, not necessary for this PR, since we're hiding UI elements (in this case the header) with this alert, we should have it fade out or auto close after a certain time